pkg/skaffold/runner/build_deploy.go

```diff
@@ -117,6 +117,8 @@ func (r *SkaffoldRunner) DeployAndLog(ctx context.Context, out io.Writer, artifa
 	if r.runCtx.Opts.PortForward.Enabled {
 		if err := r.forwarderManager.Start(ctx); err != nil {
 			logrus.Warnln("Error starting port forwarding:", err)
+		} else {
+			color.Yellow.Fprintln(out, "Press ctrl c to exit")
```

Comment:
nit:

```suggestion
			color.Yellow.Fprintln(out, "Press Ctrl+c to exit")
```

Reply:
And make that a cap C `Ctrl+C`.

(I don't know where my comment went from my review!)